scripts/install/lidarr.sh

```diff
@@ -37,40 +68,52 @@ cat > /home/${user}/.config/Lidarr/config.xml << LID
   <LaunchBrowser>False</LaunchBrowser>
 </Config>
 LID
-chown -R ${user}: /home/${user}/.config
-cat > /etc/systemd/system/lidarr.service << LID
+    mkdir -p /home/"${user}"/.tmp
+    chown -R "${user}": "/home/${user}/.tmp"
+    chown -R "${user}": "/home/${user}/.config"
+}
+systemd() {
+    cat > /etc/systemd/system/lidarr.service << LID
 [Unit]
-Description=lidarr for ${user}
+Description=Lidarr
 After=syslog.target network.target
 
 [Service]
 Type=simple
 User=${user}
 Group=${user}
 Environment="TMPDIR=/home/${user}/.tmp"
-ExecStart=/usr/bin/mono /home/${user}/Lidarr/Lidarr.exe -nobrowser
-ExecStop=-/bin/kill -HUP
+ExecStart=/opt/Lidarr/Lidarr -nobrowser -data=/home/${user}/.config/Lidarr/
 WorkingDirectory=/home/${user}/
```

Comment:
Does this need to be updated? I was under the impression that it should be the same directory as the binaries; however, if it's not needed then perhaps it should just be removed.

Reply:
I don't believe the working directory was ever nor is needed.

this is likely some legacy code that should be removed

Reply:
I mean, it works like so ha. Do we want to re-do the service file completely at that point? This is the one suggested upstream. https://wiki.servarr.com/Lidarr_Installation#Auto_Start_Using_Systemd

Reply:
We may as well just remove the `WorkingDirectory` then, I don't think there's anything beyond the TMPDIR change really, which I am fine to leave. I've only ever used the home directory specific tmp directories on multi-user installs because of permission issues, but the effect of leaving it as-is isn't really negative, as far as I can see.

If we remove working directory here the update script should also run a delete pattern on the service file as well.

Reply:
Changes pushed, resolving convo